.github/workflows/build.yml

```diff
@@ -75,6 +75,36 @@ jobs:
             echo "run_hypothesis=true" >> $GITHUB_OUTPUT
           fi
 
+  check_abi:
+    name: 'Check if the ABI has changed'
```

Comment:
```suggestion
    name: 'Check if ABI has changed'
```

Reply:
I'm not sure if this is used in the required checks list.

Reply:
It is. Also, "the ABI" is better (it is a noun, after all), so why change it?

Reply:
To make the text a bit shorter, especially helps for limited space mobile. But it doesn't matter, especially if it's a required check.